### PR TITLE
[doc] Add Uninstall @types/next in Upgrading Doc

### PR DESCRIPTION
**Pull request by azizhk**

### CODE REVIEW

File: UPGRADING.md [hunk 1]
```diff
@@ -8,6 +8,21 @@ Next.js will now ignore usage `@zeit/next-typescript` and warn you to remove it.
 
 Usage of [`fork-ts-checker-webpack-plugin`](https://github.com/Realytics/fork-ts-checker-webpack-plugin/issues) should also be removed from your `next.config.js`.
 
+Typescript Definitions are also published with next npm package. So you need to uninstall `@types/next` as they would conflict.
```
Comment (timneutkens) [inline UPGRADING.md]:
```suggestion
Typescript Definitions are published with the `next` package, so you need to uninstall `@types/next` as they would conflict.
```

File: UPGRADING.md [hunk 2]
```diff
@@ -8,6 +8,21 @@ Next.js will now ignore usage `@zeit/next-typescript` and warn you to remove it.
...
+Following types have changed: (may not be a comprehensive list, do give PRs for modifications)
```
Comment (timneutkens) [inline UPGRADING.md]:
```suggestion
The following types are different:
```

File: UPGRADING.md [hunk 3]
```diff
@@ -8,6 +8,21 @@ Next.js will now ignore usage `@zeit/next-typescript` and warn you to remove it.
...
+From:
```
Comment (timneutkens) [inline UPGRADING.md]:
```suggestion
> This list was created by the community to help you upgrade, if you find other differences please send a pull-request to this list to help other users.

From:
```

**Review by timneutkens — CHANGES_REQUESTED** [review]
Thanks! Left a few changes!

**Review by Timer — APPROVED** [review]

**Pull request merged**